Library/OcMainLib/OpenCoreUefi.c

```diff
@@ -117,19 +120,30 @@ OcLoadDrivers (
   for (Index = 0; Index < Config->Uefi.Drivers.Count; ++Index) {
     DriverEntry     = Config->Uefi.Drivers.Values[Index];
     DriverComment   = OC_BLOB_GET (&DriverEntry->Comment);
+    DriverLoad      = OC_BLOB_GET (&DriverEntry->Load);
     DriverFileName  = OC_BLOB_GET (&DriverEntry->Path);
     DriverArguments = OC_BLOB_GET (&DriverEntry->Arguments);
 
-    SkipDriver = !DriverEntry->Enabled || DriverFileName == NULL || DriverFileName[0] == '\0';
+    SkipDriver = (
+                    (DriverFileName == NULL)
+                 || (DriverFileName[0] == '\0')
+                 || (
+                     LoadEarly
+                      ? (AsciiStrCmp (DriverLoad, "Early") != 0)
+                      : (AsciiStrCmp (DriverLoad, "Enabled") != 0)
+                     )
+                    );
 
-    DEBUG ((
-      DEBUG_INFO,
-      "OC: Driver %a at %u (%a) is %a\n",
-      DriverFileName,
-      Index,
-      DriverComment,
-      SkipDriver ? "skipped!" : "being loaded..."
-      ));
+    if (!LoadEarly || !SkipDriver) {
```

Comment:
Why that particular if is used?

Reply:
Good question ... I was thinking of removing the skipped lines entirely - as there can be lots, and they perhaps don't add much (that isn't at least implicitly present anyway!) - but reasoned they can be useful all the same. But with an early and late phase of loading, you get two lots of lots of skipped lines.

I preferred to only show what is loaded in the early phase, not what is skipped, and to show what is skipped in the later/normal phase (which would include anything loaded early, at that point).

Note that if I do include these here, these come in early log (while logging is still being buffered, and when OC is writing log info to whatever default console device is present) - which has another issue for me:

 - I need as few lines as possible in this stage of early log, or my MBP10,2 (still!) hangs after the firmware update to 429.0.0.0.0 (as I already reported to you), so unfortunately for me, having a load of skipped lines here also renders by MBP10,2 more or less unbootable with OC.